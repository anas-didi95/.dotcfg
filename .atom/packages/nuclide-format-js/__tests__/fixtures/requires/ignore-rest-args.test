function foo(...a) {
  a;
  b;
}

(function(...c) {
  c;
  d;
})();

((...e) => {
  e;
  f;
})();
