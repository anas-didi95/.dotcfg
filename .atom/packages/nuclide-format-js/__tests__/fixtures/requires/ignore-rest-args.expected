const b = require('b');
const d = require('d');
const f = require('f');

function foo(...a) {
  a;
  b;
}

(function(...c) {
  c;
  d;
})();

((...e) => {
  e;
  f;
})();
